test/Cache-test.js

```diff
@@ -177,5 +178,20 @@ describe('raptor-cache' , function() {
             ],
             done);
     });
-});
 
+    // Promise tests that will only run on Node 4+
+    if (promisesExist) {
```

Comment:
Personally, I think we should drop support for Node < 4, so we can properly add tests for the Promise code paths.